src/useMultiRootEditor.tsx

```diff
@@ -113,54 +120,56 @@ const useMultiRootEditor = ( props: MultiRootHookProps ): MultiRootHookReturns =
 	const onChangeData = ( editor: MultiRootEditor, event: EventInfo ): void => {
 		const modelDocument = editor!.model.document;
 
-		const newData: Record<string, string> = {};
-		const newAttributes: Record<string, Record<string, unknown>> = {};
+		if ( !props.disableTwoWayDataBinding ) {
```

Comment:
Probably incorrect `props` object reference, which may lead to outdated `disableTwoWayDataBinding` flag value. Please check what happens if we re-render component with toggled `disableTwoWayDataBinding` _after_ initialization of editor. It'd be nice to add a test that checks this scenario.

Reply:
The config provided to the `useMultiRootEditor` should be static and should not be changed during the editor's life. Why is it incorrect?

Reply:
@DawidKossowski If so, then it's not an issue, and we can stay as it is. Some of the attributes[1] passed using `props` argument can be modified between re-renders, though.

[1] https://github.com/ckeditor/ckeditor5-react/blob/7b7c869b8179fb0b15ce1c48d0066ed89d774468/src/useMultiRootEditor.tsx#L65
[2] https://github.com/ckeditor/ckeditor5-react/blob/7b7c869b8179fb0b15ce1c48d0066ed89d774468/src/useMultiRootEditor.tsx#L75